code/modules/events/disease_outbreak.dm

```diff
@@ -1,41 +1,75 @@
+GLOBAL_VAR(current_pending_disease)
 /datum/event/disease_outbreak
-	announceWhen = 180
 	/// The type of disease that patient zero will be infected with.
 	var/datum/disease/D
-	/// The initial target of the disease.
-	var/mob/living/carbon/human/patient_zero
+
+	var/list/disease_blacklist = list(/datum/disease/advance, /datum/disease/appendicitis, /datum/disease/kuru, /datum/disease/critical, /datum/disease/rhumba_beat, /datum/disease/fake_gbs,
+										/datum/disease/gbs, /datum/disease/transformation, /datum/disease/food_poisoning, /datum/disease/advance/cold, /datum/disease/advance/flu, /datum/disease/advance/heal,
+										/datum/disease/advance/hullucigen, /datum/disease/advance/sensory_restoration, /datum/disease/advance/voice_change)
+	var/static/list/transmissable_symptoms = list()
+	var/static/list/diseases_minor = list()
+	var/static/list/diseases_moderate = list()
+	var/static/list/diseases_major = list()
 
 /datum/event/disease_outbreak/setup()
+	if(isemptylist(diseases_minor) && isemptylist(diseases_moderate) && isemptylist(diseases_major))
+		populate_diseases()
+	if(isemptylist(transmissable_symptoms))
+		populate_symptoms()
+	var/datum/disease/virus
 	if(prob(25))
-		var/virus_type = pick(/datum/disease/advance/flu, /datum/disease/advance/cold, /datum/disease/brainrot, /datum/disease/magnitis, /datum/disease/beesease, /datum/disease/anxiety, /datum/disease/fake_gbs, /datum/disease/fluspanish, /datum/disease/pierrot_throat, /datum/disease/lycan)
-		D = new virus_type()
+		switch(severity)
+			if(EVENT_LEVEL_MUNDANE) virus = pick(diseases_minor)
+			if(EVENT_LEVEL_MODERATE) virus = pick(diseases_moderate)
+			if(EVENT_LEVEL_MAJOR) virus = pick(diseases_major)
```

Comment:
Please don't put multiple expressions on the same line
```suggestion
			if(EVENT_LEVEL_MUNDANE) 
				virus = pick(diseases_minor)
			if(EVENT_LEVEL_MODERATE) 
				virus = pick(diseases_moderate)
			if(EVENT_LEVEL_MAJOR) 
				virus = pick(diseases_major)
```

You might also want to add a default case with a stack trace in the event of an invalid event level (or just a default virus), otherwise virus() might be null on init.